Client/Frontend/Library/LibraryViewController/LibraryViewController+LibraryPanelDelegate.swift

```diff
@@ -6,38 +6,15 @@ import Storage
 import Shared
 
 extension LibraryViewController: LibraryPanelDelegate {
-    func libraryPanelDidRequestToSignIn() {
-        self.dismiss(animated: false, completion: nil)
-        delegate?.libraryPanelDidRequestToSignIn()
-    }
-
-    func libraryPanelDidRequestToCreateAccount() {
-        self.dismiss(animated: false, completion: nil)
-        delegate?.libraryPanelDidRequestToCreateAccount()
-    }
-
     func libraryPanelDidRequestToOpenInNewTab(_ url: URL, isPrivate: Bool) {
         delegate?.libraryPanelDidRequestToOpenInNewTab(url, isPrivate: isPrivate)
+        DispatchQueue.main.async {
+            self.dismiss(animated: true, completion: nil)
+        }
```

Comment:
Why are we now dismissing from here? If that's needed for coordinators, I would expect we dismiss with the coordinator